users/views.py

```diff
@@ -1,3 +1,17 @@
 from django.shortcuts import render
+from django.http import HttpResponse
 
-# Create your views here.
+from users.forms import RegistrationForm
+
+
+def register(request):
+    if request.method == 'POST':
+        form = RegistrationForm(request.POST)
+        if form.is_valid():
+            form.save()
+
+            return HttpResponse("You successfully created a new account!")
+    else:
+        form = RegistrationForm()
+
+    return render(request, 'registration.html', {'form': form})
```

Comment:
Method views are ok for creating general purpose endpoints with no strict ties to models, templates or web standards.
Django provides a better way to expose view endpoints in your application. You can use one of the Django view classes. They will allow you to DRY data consistently.

Depending on whether you want to tie the view with a Model for CRUD purposes, Template for presentation purposes or simply creating an API (e. g. REST API) endpoints that serve or receive data, you'll choose one of the specific View classes.

Give it a try.